about.html

```diff
@@ -10,55 +10,67 @@
   <link rel="stylesheet" href="css/index.css">
 
 </head>
-
-<body>
+<div class="top-container">
+  <header>
+      <nav>
+          <img src="img/lambda-black.png" alt=""> 
```

Comment:
Don't forget to fill out your href.  A user cannot get to the home or about page.